### ISSUE TITLE
warning java still!!!

### ISSUE THREAD

**Issue by Ultimate132** [Jire/Charlatano#988]
With this new update 1.1.4 it still appear the warning and this time no resource is working!
https://prnt.sc/wcn15r
Here is the print I still need to use the version of July :(

**Comment by apidotmy** [comment]
your path bruh

**Comment by Ultimate132** [comment]
???

**Comment by Jire** [comment]
Those warnings shouldn't effect the usability.

**Comment by Ultimate132** [comment]
But it affects no features are working im still using the version of july !
That is working fine

**Comment by Jire** [comment]
Let me know if fixed in [1.1.5](https://github.com/Jire/Charlatano/releases/tag/1.1.5)

**Comment by Ultimate132** [comment]
Yeah it is fixed thanks bro it appear the warnings but it works fine!
Thank You!
